modules/olmv1-catalog-plain.adoc

```diff
@@ -0,0 +1,244 @@
+endif::[]
+
+:_content-type: PROCEDURE
+
+[id="olmv1-building-plain-bundle-image-source_{context}"]
+= Building a plain bundle image from an image source
+
+The Operator Controller currently supports installing plain bundles created only from a _plain bundle image_.
+
+.Prequisites
+
+* You have Kubernetes manifests for your bundle in a flat directory at the root of your project similar to the following structure:
++
+.Example directory structure
+[source,terminal]
+----
+manifests
+├── namespace.yaml
+├── service_account.yaml
+├── cluster_role.yaml
+├── cluster_role_binding.yaml
+└── deployment.yaml
+----
+
+.Procedure
+
+. At the root of your project, create a Dockerfile that can build a bundle image:
++
+.Example `plainbundle.Dockerfile`
+[source,docker]
+----
+FROM scratch <1>
+ADD manifests /manifests
+----
+<1> Use the `FROM scratch` directive to make the size of the image smaller. No other files or directories are required in the bundle image.
+
+. Build an OCI-compliant image using your preferred build tool, similar to the following example:
++
+[source,terminal]
+----
+$ podman build -f plainbundle.Dockerfile -t \
+    quay.io/<organization_name>/<repository_name>:<image_tag> . <1>
+----
+<1> Use an image tag that references a repository where you have push access privileges.
+
+. Push the image to your remote registry:
++
+[source,terminal]
+----
+$ podman push quay.io/<organization_name>/<repository_name>:<image_tag>
+----
+
+[id="olmv1-creating-fbc_{context}"]
+= Creating a file-based catalog
+
+If you do not have a file-based catalog, you must perform the following steps to initialize the catalog.
+
+.Procedure
+
+. Create a directory for the catalog by running the following command:
++
+[source,terminal]
+----
+$ mkdir <catalog_dir>
+----
+
+. Generate a Dockerfile that can build a catalog image by running the `opm generate dockerfile` command in the same directory level as the previous step:
++
+[source,terminal,subs="attributes+"]
+----
+ifdef::openshift-origin[]
+$ opm generate dockerfile <catalog_dir>
+endif::[]
+ifndef::openshift-origin[]
+$ opm generate dockerfile <catalog_dir> \
+    -i {registry-image} <1>
+endif::[]
+----
+ifndef::openshift-origin[]
+<1> Specify the official Red Hat base image by using the `-i` flag, otherwise the Dockerfile uses the default upstream image.
+endif::[]
++
+[NOTE]
+====
+The generated Dockerfile must be in the same parent directory as the catalog directory that you created in the previous step:
+
+.Example directory structure
+[source,terminal]
+----
+.
+├── <catalog_dir>
+└── <catalog_dir>.Dockerfile
+----
+====
+
+. Populate the catalog with the package definition for your extension by running the `opm init` command:
++
+[source,terminal]
+----
+$ opm init <extension_name> \
+    --output json \
+    > <catalog_dir>/index.json
+----
++
+This command generates an `olm.package` declarative config blob in the specified catalog configuration file.
```

Comment:
(Unless this is normal terminology for this context)
```suggestion
This command generates an `olm.package` declarative configuration blob in the specified catalog configuration file.
```

Reply:
Normally I would per IBM SG, but leaving "config blob"s alone for now to match `opm` CLI help output elsewhere. Will need to address in the CLI before potentially switching/syncing up to "configuration blob".